src/AspNetCoreModuleV2/RequestHandlerLib/PipeOutputManager.cpp

```diff
@@ -40,15 +49,17 @@ PipeOutputManager::StopOutputRedirection()
 
     if (m_hErrThread != NULL &&
         m_hErrThread != INVALID_HANDLE_VALUE &&
-        GetExitCodeThread(m_hErrThread, &dwThreadStatus) != 0 &&
+        LOG_LAST_ERROR_IF(GetExitCodeThread(m_hErrThread, &dwThreadStatus) != 0) &&
```

Comment:
You need to log when result is 0

Reply:
Yeah, these checks are incorrect right now.